### ISSUE TITLE
onMenuSwipe

### ISSUE THREAD

**Issue by tsarma** [adgsm/multi-level-push-menu#110]
Hi there, Is there a to detect if a menu is opening or closing, with a callback funtion "onMenuSwipe".

**Comment by adgsm** [comment]
Not sure to be honest, I would have to look closer into the source code... However, can't you use onCollapseMenuStart and onExpandMenuStart to set flags?

**Comment by tsarma** [comment]
Yes I may use onCollapseMenuStart and onExpandMenuStart, but not in a way I wanted it. Because these two flags are trigger on every level. 

What I actually looking for is, a way to detect if menu is once opened or complete closed. A flag/callback should trigger only once on first level open or close, not every time. Why I wanted this is, because I have a trigger button which should be styled different when my menu (with 3 levels) is opened then when it is closed.

thank you for the response

**Comment by WriterStat** [comment]
If you are still looking for an answer, I believe you could use adgsm's suggestion, plus the api code below inside them.

var open = $('#menu').multilevelpushmenu('activemenu');

It returns false if collapsed and the level if not. You could easily then check if open == false or if open == 1/level 1. Yes it would trigger/check on all, but it's not a menu drain or drain on system to speak of in any way really and still useful.  

And style the button different dependent on this. Unless I mistook this.

Note: I ran across this because I needed a way of changing the push objects div widths and resizing them depending on the changed size of the menu. Knowing the active menu, allowed me to calculate the menu's width dynamically. Hope this is helpful.
